### ISSUE TITLE
about convergence speed of training process

### ISSUE THREAD

**Issue by lynlindasy** [PRIS-CV/Fine-Grained-or-Not#2]
Hello, I tried your method on CUB Bird but found the printed loss is different from your log in this repo. Is there any details have been changed? Thank you.

**Comment by dongliangchang** [comment]
no

**Comment by shenglih** [comment]
hi I tried the stanford car example, the gradient kept exploding... (train_loss = nan, test_loss = nan)... i tried some fixes but none worked... Do you happen to know why by any chance? Thanks so much

**Comment by dongliangchang** [comment]
@shenglih 
1. Please check the version of the PyTorch. PyTorch 2 and 3 have big differences in the loss function
2. Please check the folder name, the folder name should be 001,002,003,... instead of 1,2,3,...

**Comment by shenglih** [comment]
Thank you so much for your prompt reply!!! Changing the folder name worked!! LOL

**Comment by shenglih** [comment]
Sorry to bug again... but it weirdly worked once but whenever I restarted it remained nan again.... I used python 3.6, torch 1.2.0 and torchvision 0.4.0... Any other clues by any chance? Thanks so much Dongliang!

----UPDATE: ALL GOOD I GOT IT TO WORK!! THANK YOU SO MUCH
